scripts/lib/api/mergeClassMembers.ts

```diff
@@ -62,17 +63,30 @@ export async function mergeClassMembers(
           .use(remarkGfm)
           .use(remarkMath)
           .use(() => {
-            return async (tree: Root) => {
-              for (const node of tree.children) {
+            return async (root: Root) => {
+              // The attribute and method's section can be found under the class component
+              const mdxClassElement = root.children
+                .filter(
+                  (node): node is MdxJsxFlowElement =>
+                    node.type == "mdxJsxFlowElement" && node.name == "Class",
+                )
+                .shift();
+
+              for (const node of mdxClassElement?.children ?? []) {
                 await replaceMembersAfterTitle(
-                  tree,
+                  mdxClassElement!,
```

Comment:
You can make the types and code more clear by early returning if `mdxClassElement` is undefined rather than entering this for loop